F/Fontconfig/build_tarballs.jl

```diff
@@ -21,18 +21,20 @@ apk add gperf
 # Ensure that `${prefix}/include` is..... included
 export CPPFLAGS="-I${prefix}/include"
 
+FLAGS=()
 if [[ "${target}" == *-linux-* ]] || [[ "${target}" == *-freebsd* ]]; then
-    FONTS_DIR="/usr/local/share/fonts"
+    FLAGS+=(--with-add-fonts="/usr/local/share/fonts")
+    FLAGS+=(--with-cache-dir="/var/cache/fontconfig")
```

Comment:
FYI this path doesn't exist on FreeBSD: it's `/var/db/fontconfig`.

Reply:
Thank you very much!